fe/fe-core/src/main/java/com/starrocks/credential/aws/AWSCloudConfiguration.java

```diff
@@ -80,6 +86,35 @@ public void applyToConfiguration(Configuration configuration) {
         awsCloudCredential.applyToConfiguration(configuration);
     }
 
+    @Override
+    public void loadCommonFields(Map<String, String> properties) {
+        super.loadCommonFields(properties);
+        enablePartitionedPrefix = Boolean.parseBoolean(
+                properties.getOrDefault(CloudConfigurationConstants.AWS_S3_ENABLE_PARTITIONED_PREFIX, "false"));
+        String value = properties.get(CloudConfigurationConstants.AWS_S3_NUM_PARTITIONED_PREFIX);
+        if (enablePartitionedPrefix) {
+            this.numOfPartitionedPrefix = DEFAULT_NUM_OF_PARTITIONED_PREFIX;
+            if (value != null) {
+                try {
+                    int val = Integer.parseInt(value);
+                    if (val < 0) {
```

Comment:
deal with 0

Reply:
0 is dealed as if the value is not provided, which is handled in line 96.